lib/cli/run.js

```diff
@@ -369,7 +369,12 @@ exports.handler = async function (argv) {
   try {
     await runMocha(mocha, argv);
   } catch (err) {
-    console.error('\n' + (err.stack || `Error: ${err.message || err}`));
+    if (!err) {
+      console.error('\n Undefined error:', err);
+    } else {
+      console.error('\n' + (err.stack || `Error: ${err.message}`), err);
```

Comment:
```suggestion
          console.error('\n' + (err.stack || `Error: ${err.message || err}`));
```
Meant to have it work as the original